src/main/webapp/index.xhtml

```diff
@@ -1,11 +1,18 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:ui="http://java.sun.com/jsf/facelets"
+      xmlns:h="http://java.sun.com/jsf/html"
+>
+<h:head>
     <title>ToolRent</title>
-</head>
-<body>
-<h2>"Hello World!</h2>
-<h3>Would you like to rent something? :)))</h3>
-<!--<a href="hello-servlet">Hello Servlet</a>-->
-</body>
+</h:head>
+<h:body>
+    <ui:insert name="navbar">
+        <ui:include src="templates/navbar.xhtml"/>
+    </ui:insert>
+
+    <h2>"Hello World!</h2>
+    <h3>Would you like to rent something? :)))</h3>
+    <!--<a href="hello-servlet">Hello Servlet</a>-->
```

Comment:
Maybe we should delete commented out code?
Or was it left out there for a reason?

Reply:
Yes, thanks for noticing